pkg/metrics/exporter/export.go

```diff
@@ -40,32 +40,33 @@ func WithOutputFile(metricsFile string) Opt {
 	}
 }
 
-func NewExporter(opts ...Opt) (*Exporter, error) {
+func NewExporter(opts ...Opt) error {
```

Comment:
Better to initialize the exporter in the package init() function which guarantees that it must be successfully initialized.
Then RecordDaemonEvent() can't fail which makes it neater without checking its results. we can ignore its absence.

Reply:
Done.